### PR TITLE
Update aws-singletier

### PR DESCRIPTION
**Pull request by jdmooney**
Update single-tier doc to reflect changes in AWS procedures

### CODE REVIEW

File: userguide/tutorials/aws-singletier.adoc [hunk 1]
```diff
@@ -109,11 +103,11 @@ You will now be able to explore your instance and perform various configuration
 
 == Testing
 
-You can now try to login to KAUI from your browser using the URL http://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the KAUI login screen. If this does *not* work, try https://INSTANCE_IP:443 (actually the :443 can be omitted). The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
+You can now try to login to KAUI from your browser using the URL https://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the KAUI login screen. The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
```
Comment (ghost) [inline userguide/tutorials/aws-singletier.adoc:106]: "The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem."

Is this still true even though they are now using "https"?

**Review by ghost — APPROVED** [review]
Just one question/comment.

File: userguide/tutorials/aws-singletier.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/tutorials/aws-singletier.adoc:106]: This is correct, unless a fully valid certificate is already in place.

Comment (ghost) [reply userguide/tutorials/aws-singletier.adoc:106]: 👍

**Pull request merged**